Utils/gitlab_triggers/trigger_content_nightly_build.sh

```diff
@@ -12,17 +12,19 @@ if [ "$#" -lt "1" ]; then
   echo "Usage:
   $0 -ct <token>
 
-  [-ct, --ci-token]           The ci gitlab trigger token.
-  [-b, --branch]              The branch name. Default is the current branch.
-  [-ch, --slack-channel]      A slack channel to send notifications to. Default is dmst-build-test.
-  [-s, --sdk-ref]             The sdk ref to use. Default is the latest nightly.
+  [-ct, --ci-token]                         The ci gitlab trigger token.
+  [-b, --branch]                            The branch name. Default is the current branch.
+  [-ch, --slack-channel]                    A slack channel to send notifications to. Default is dmst-build-test.
+  [-s, --sdk-ref]                           The sdk ref to use. Default is the latest nightly.
+  [-tmr --test-modeling-rule-jira-tickets]  Whether to test modeling rule jira tickets. Default is false.
```

Comment:
```suggestion
  [-tmr --test-modeling-rule-jira-tickets]  Whether to test modeling rule jira tickets. Possible values: true, false. Default is false.
```

Reply:
this is a bool flag. no value should be provided by the user. fixed.